apps/web/src/components/AccountSelectorModal/AccountSelectorModal.tsx

```diff
@@ -60,7 +60,7 @@ export const AccountSelectorModal = () => {
                   paddingLeft="12px"
                 >
                   <Heading color={color("900")} size="sm">
-                    {capitalize(type)}
+                    {type.split("_").map(capitalize).join(" ")}
```

Comment:
Maybe it would be better to move it to a separate variable/function?

Reply:
created a [ticket](https://app.asana.com/0/1207041888417083/1207877246316773/f) for that